contracts/test/sdk/IpcContract.t.sol

```diff
@@ -130,12 +142,19 @@ contract IpcExchangeTest is Test {
         CallMsg memory lastCall = exch.getLastCallMsg();
         require(keccak256(abi.encode(envelope)) == keccak256(abi.encode(lastEnvelope)), "unexpected envelope");
         require(keccak256(abi.encode(callMsg)) == keccak256(abi.encode(lastCall)), "unexpected callmsg");
+    }
 
+    function test_IpcExchangeFlipRevert() public {
```

Comment:
All of these tests are related to `IpcExchange`. Can we drop `IpcExchange` in tests' names and name them eg. `test_FlipRevert`?

Reply:
Agree that the verbose-ness is helpful here. Also when running the full test suite it s nice to see a long and descriptive test name for failures